packages/renderer/src/lib/image/ImagesList.svelte

```diff
@@ -214,6 +173,32 @@ function computeInterval(): number {
   // every day
   return 60 * 60 * 24 * SECOND;
 }
+
+let selectedItemsNumber: number;
+let table: Table;
+
+let statusColumn = new Column<ImageInfoUI>('Status', 'center', '70px', ImageColumnStatus);
```

Comment:
I think we'll reach soon a limit on constructor parameters

so while the name of the column looks quite important, I think the other parameters are kind of optional

so either we move to dedicated method or we provide JSON like parameters with an interface so it'll easier to grow/add mandatory parameters/optional parameters

```ts
new Column<ImageInfoUI>('Status', {align: 'center', width: '70px'});
```
or 

```ts
new Column<ImageInfoUI>('Status')
column.width = '70px';
column.align = 'center';
```

or use fluent API like
```ts
Column.name('Status').width('70px').align('center').create();
```

Reply:
I went with the first option in the latest commit. It does complicate things a bit because it makes sense to have most of the parameters optional, but that makes more sense and keeps the use simpler.